practice-three/README.md

```diff
@@ -0,0 +1,43 @@
+# Design a responsive website
+> A simple website.
+
+Build a site to practice frontend during HTML/CSS training.
+
+## Getting Started 
+
+### Figma design
+
+Design a website based on Figma's sample design: [here](https://www.figma.com/file/NWlbUjPg4GynhdQcBvqTdw/Responsive-Web-Page-%F0%9F%8E%89-(Community)?node-id=102%3A191)
+
+### Prerequisites
+
+Before you continue, ensure you meet the following requirements:
+* You have installed the latest VScode.
+* Use the latest browser version (Chrome, Firefox, MS Edge)
+
+### Requirement
+* Work cross devices, cross browser
+
+### Installation
+* You can download my code [here](https://github.com/jinety/practice-html-css)
+
+### Technologies
+* HTML5
+* CSS3
+* Parcel
+
+### Executing program
+* Step 1: clone repository <br>
+  `git clone https://github.com/jinety/practice-html-css/tree/feature/practice-three`
+* Step 2: cd folder practice-1 <br>
+  `cd practice-three`
+* Step 3: Setting Parcel <br>
+  `npm install --save-dev parcel`
+* Step 4: Run project <br>
+  `npx parcel src/index.html`
```

Comment:
Try to set up this into `scripts` in package.json file. Sample: https://nodejs.dev/learn/the-package-json-guide